api/v1alpha4/metal3machine_types.go

```diff
@@ -62,6 +62,11 @@ type Metal3MachineSpec struct {
 	// NetworkData is an object storing the reference to the secret containing the
 	// network data given by the user.
 	NetworkData *corev1.SecretReference `json:"networkData,omitempty"`
+	// When set to disabled, automated cleaning will be skipped
```

Comment:
cleaning of what ? disk ? which disk (excluding boot disk ?)

Reply:
done